app/controllers/api/v1/admin/site_settings_controller.rb

```diff
@@ -30,7 +30,13 @@ def update
           return render_error status: :not_found unless site_setting
 
           update = if params[:name] == 'BrandingImage'
-                     site_setting.image.attach params[:site_setting][:value]
+                     # Case where user is removing the custom branding image
+                     if params[:site_setting][:value].nil?
+                       site_setting.image.purge
```

Comment:
Can you try replacing this with `site_setting.image.attach ActionController::Base.helpers.image_path('bbb_logo.png')`

Reply:
Yeah this is what I initially tried to do but it wasn't working due to the image_path not getting the correct path. I tried other helper methods to get the correct path, but the only alternative to what I did that worked was this:
`site_setting.image.attach(io: File.open('/home/blindsidenetworks/greenlight/app/assets/images/bbb_logo.png'), filename: 'bbb_logo.png')`, which I'm not sure is better. Let me know what you think, also let me know what you think about creating it's own endpoint for this if I keep the `purge`

Reply:
Yeah I think creating a separate endpoint then would make more sense. The logic here is becoming a bit much